.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: php-actions/composer@v6
+        with:
+          # Lowest supported PHP version
+          php_version: "7.0"
```

Comment:
Ik kreeg bij het testen de foutmelding dat de composer versie niet compatibel is met php7.0. PHP 7.4. werkte wel.
````
php_version: "7.4"
```
Wat mij betreft is ondersteuning vanaf php 7.4 voldoende.